pkg/operation/botanist/matchers/matcher.go

```diff
@@ -164,10 +163,6 @@ var (
 		{GVR: certificatesv1.SchemeGroupVersion.WithResource("certificatesigningrequests"), ClusterScoped: true, Subresource: "status"},
 		{GVR: certificatesv1.SchemeGroupVersion.WithResource("certificatesigningrequests"), ClusterScoped: true, Subresource: "approval"},
 
-		{GVR: certificatesv1beta1.SchemeGroupVersion.WithResource("certificatesigningrequests"), ClusterScoped: true},
-		{GVR: certificatesv1beta1.SchemeGroupVersion.WithResource("certificatesigningrequests"), ClusterScoped: true, Subresource: "status"},
-		{GVR: certificatesv1beta1.SchemeGroupVersion.WithResource("certificatesigningrequests"), ClusterScoped: true, Subresource: "approval"},
-
```

Comment:
I think this is still needed since we support shoot clusters with Kubernetes 1.17 which don't have the `v1` API yet. Please add a comment that this can be cleaned up once support for support for < 1.19 is dropped (here the `v1` version was introduced).